src/Resources/public/js/pimcore/settingsMeta.js

```diff
@@ -307,13 +311,13 @@ valantic.dataquality.settings_meta = Class.create({
         const formPanel = new Ext.form.FormPanel({
             bodyStyle: 'padding:10px;',
             // eslint-disable-next-line max-len
-            items: [classnameCombo, localeCombo, greenRange, orangeRange, nestingLimitRange, ignoreFallbackLanguage, disableTabOnObject],
+            items: [classnameCombo, localeInfo, localeCombo, greenRange, orangeRange, nestingLimitRange, ignoreFallbackLanguage, disableTabOnObject],
         });
 
         const modifyWin = new Ext.Window({
             modal: true,
-            width: 300,
-            height: 470,
+            width: 380,
+            height: 600,
```

Comment:
```suggestion
            height: 620,
```

Currently the last row of text is slightly cut off

<img width="412" alt="image" src="https://github.com/valantic/pimcore-data-quality-bundle/assets/474329/40dee481-adbc-4f8e-a735-e0a05e30eeef">